### PR TITLE
DOGE-22

### PR DESCRIPTION
**Pull request by remingtron**
- Added packer steps to build pipeline (these were missing). Only build a packer image if the manifest.json file is not present. If this occurs on master, we also commit this file back to the repository.
  - This also caused us to add the workspace clean step
- Deploying dev task service with terraform: downloading the war from artifactory and restarting tomcat.
- Removed docker from task service.

### CODE REVIEW

File: packer/build [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -e
+
+pushd `dirname "$0"`
+
+if [ -e manifest.json ]; then
```
Comment (duraz0rz) [inline packer/build:5]: If `packer.json` changes, will Packer still run correctly? Now that there is an `update_git` script, where do we delete `manifest.json` to make sure Packer runs on changes??

Comment (remingtron) [reply packer/build:5]: This is something you would manually do as part of a pull request that requires a new image. We stole the pattern from https://github.com/PillarTechnology/PillarWebReverseProxy

Comment (duraz0rz) [reply packer/build:5]: Might want to call that out in the readme then.

Comment (remingtron) [reply packer/build:5]: Good call

**Review by duraz0rz — APPROVED** [review]
:shipit:

**Pull request merged**